src/site/content/en/learn/css/media-queries/index.md

```diff
@@ -0,0 +1,786 @@
+
+### Media Types
+
+Media types correspond to broad categorizations of devices.  The following are prevalent or well-supported media types browser supports.
+
+* **all**: All media
+
+* **screen**: Media that has rendered the contents of the document the styles are applied to with a screen
+
+* **print**: Media that has rendered the contents of the document the styles in a way that’s optimized for print temporarily (I.e. print preview) or permanently.
+
+* **speech**: Media that can voice the contents of the document the styles are applied to audibly
+
+The `all` media type can be omitted in most queries you write that isn't concerned with what type of media a medium is because the `all`  media type is used by default in media queries that don’t require a media type specified:
+
+{% Compare 'better' %}
+```css
+@media all and (min-width: 31.25em) {}
+```
+{% CompareCaption %}
+You can elect to write queries with a media type always specified with `all` being the broadest and default one used by browsers…
+{% endCompareCaption %}
+{% endCompare %}
+
+{% Compare 'better' %}
+```css
+/* Equivalent to */
+@media (min-width: 31.25em) {}
+```
+{% CompareCaption %}
+Alternatively you can omit the `all` media type to save a few characters of typing if `all` –the default media type used by browsers–fits your media query needs
+{% endCompareCaption %}
+{% endCompare %}
+
+{% Aside %}
+With this default handling of the media type associated with media queries, most queries in this module will usually omit specififying a media type unless it's needed to be something other than the `all` media type. This aligns wth how media queries are typically used throughout the Web.
+{% endAside %}
+
+#### Querying whether a medium is a particular type
+A valid media query can be as plain as a valid media type such as the following example where a document with a deliberately dark background is changed to be light-themed to save user’s ink by querying if the media the document is being consumed in can be considered a print-based medium:
+
+```css
+@media print {
+  --background-color: transparent;
+  --primary-color: black;
+}
+```
+
+The demo below demonstrates these styles conditionally applied in print-media specific contexts:
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'OJxygKv',
+  theme: 'dark',
+  height: 700,
+  tab: 'css,result'
+} %}
+
+{% YouTube 'VLhGBT68N1k', {loop: "1"} %}
+
+#### Querying whether a medium is not a particular type
+Using the aforementioned `not` query modifier keyword, you are able to conditionally apply styles if a medium isn’t a particular media type by prefixing the media type with the keyword; the previous example’s page styles specific to non-print environments can be grouped within a media query that conditionally applies the styles only if the medium isn’t print-based:
+```css
+
+@media not print {
+  /* Styles to conditionally apply if the medium can't be categorized being associated with print-based media */
+}
+```
```

Comment:
```suggestion
#### Querying whether a medium is not a particular type

Using the aforementioned `not` query modifier keyword, you are able to conditionally apply styles if a medium isn’t a particular media type by prefixing the media type with the keyword; the previous example’s page styles specific to non-print environments can be grouped within a media query that conditionally applies the styles only if the medium isn’t print-based:

```css
@media not print {
  /* Styles to conditionally apply if the medium can't be categorized being associated with print-based media */
}
```
```